modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -1080,10 +1080,22 @@ class Editor implements EditorObservable {
     VisualAids.addVisual(this, elm);
   }
 
+  /**
+   * Changes the editable state of the editor root element.
```

Comment:
```suggestion
   * Changes the editable state of the editor’s root element.
```

Inflect for the possessive case with the Saxon genitive.

Reply:
Changed this. Thanks for noticing this.